google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -102,6 +106,21 @@ interfaces:
     field_name_patterns:
       subscription: subscription
     timeout_millis: 60000
+  - name: UpdateSubscription
+    flattening:
+      groups:
+      - parameters:
+        - subscription
+        - update_mask
+    required_fields:
+    - subscription
+    - update_mask
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    field_name_patterns:
+      subscription.name: subscription
```

Comment:
Simply removing the field name patterns.

Tracking this bug here: https://github.com/googleapis/toolkit/issues/1060